scripts/ci-tool.py

```diff
@@ -249,6 +249,12 @@ def test_project(build_root, include_regex=None, exclude_regex=None):
         '-T', TEST_NAME
     ]
 
+    # Need to convert test names from set to a regex of ORs
+    names_as_list = list(test_names)
+    names_as_regex = "|".join(str(x) for x in names_as_list)
```

Comment:
You don't need the list here, do you? You can iterate over `test_names` directly.

Reply:
you're right